src/components/Reports/ReportEdit.tsx

```diff
@@ -293,7 +293,6 @@ export default function ReportEdit({ organization }: ReportEditProps): JSX.Eleme
         await updatePhotos(report.id);
         const submitResult = await ReportService.submitReport(reportIdInt);
         if (submitResult.requestSucceeded) {
-          await ReportService.unlockReport(reportIdInt);
```

Comment:
at this point is it already locked? if so, maybe BE should unlock but without updating the status?

Reply:
It gets unlocked in the submit call. This should have been a no-op, but it was changing the status

Reply:
got it